src/main/java/com/blackducksoftware/integration/hub/model/view/VersionBomComponentView.java

```diff
@@ -48,60 +35,24 @@
 import com.blackducksoftware.integration.hub.model.view.components.ReviewedDetailsView;
 import com.blackducksoftware.integration.hub.model.view.components.RiskProfileView;
 import com.blackducksoftware.integration.hub.model.view.components.VersionBomLicenseView;
-import com.google.gson.annotations.SerializedName;
 
 public class VersionBomComponentView extends HubView {
-    @SerializedName("activityData")
-    public ActivityDataView activityData = null;
-
-    @SerializedName("activityRiskProfile")
-    public RiskProfileView activityRiskProfile = null;
-
-    @SerializedName("component")
-    public String component = null;
-
-    @SerializedName("componentName")
-    public String componentName = null;
-
-    @SerializedName("componentVersion")
-    public String componentVersion = null;
-
-    @SerializedName("componentVersionName")
-    public String componentVersionName = null;
-
-    @SerializedName("licenseRiskProfile")
-    public RiskProfileView licenseRiskProfile = null;
-
-    @SerializedName("licenses")
-    public List<VersionBomLicenseView> licenses = new ArrayList<>();
-
-    @SerializedName("operationalRiskProfile")
-    public RiskProfileView operationalRiskProfile = null;
-
-    @SerializedName("origins")
+    public ActivityDataView activityData;
+    public RiskProfileView activityRiskProfile;
+    public String component;
+    public String componentName;
+    public String componentVersion;
+    public String componentVersionName;
+    public RiskProfileView licenseRiskProfile;
+    public List<VersionBomLicenseView> licenses;
```

Comment:
This is going to be a problem if we have code that expects licenses to be an empty list and not null